recipe/meta.yaml

```diff
@@ -1,77 +1,76 @@
 source:
-    url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-    sha256: d3363bb5df72d74420986a435449f3c3979285941dff57d5d97ecba352a0e3e2
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: a650525303ee866fb0c62c82f68681fcc2183eebbfafae552c27d30125fe518b
 
 build:
-    noarch: python
-    number: 1
-    script: "{{ PYTHON }} -m pip install . --no-deps -vv"
-    entry_points:
-        - celery = celery.__main__:main
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  entry_points:
+    - celery = celery.__main__:main
 
 requirements:
-    build:
-        - python
-        - pip
-    run:
-        - python
-        - pytz
-        - billiard >=3.6.1,<4.0
-        - kombu >=4.6.7,<5.0
-        - vine >=1.3.0
-        - sqlalchemy
-        - cryptography
+  build:
+    - python
+    - pip
+  run:
+    - python
+    - pytz
+    - billiard >=3.6.3,<4.0
+    - kombu >=4.6.8,<4.7
+    - sqlalchemy
+    - cryptography
 
 test:
-    requires:
-        - case >=1.3.1
-        - pytest >=3.0
-    source_files:
-        - t
-    imports:
-        - celery
-        - celery.app
-        - celery.apps
-        - celery.backends
-        - celery.backends.database
-        - celery.bin
-        - celery.concurrency
-        - celery.contrib
-        - celery.events
-        - celery.fixups
-        - celery.loaders
-        - celery.security
-        - celery.task
-        - celery.utils
-        - celery.utils.dispatch
-        - celery.worker
-    commands:
-        - celery --version
-        - celery worker --help
-        - celery beat --help
-        - celery multi help
-        - celery shell --help
-        - pytest t/unit  # [py3k]
+  requires:
+    - case >=1.3.1
+    - pytest >=3.0
+  source_files:
+    - t
+  imports:
+    - celery
+    - celery.app
+    - celery.apps
+    - celery.backends
+    - celery.backends.database
+    - celery.bin
+    - celery.concurrency
+    - celery.contrib
+    - celery.events
+    - celery.fixups
+    - celery.loaders
+    - celery.security
+    - celery.task
+    - celery.utils
+    - celery.utils.dispatch
+    - celery.worker
+  commands:
+    - celery --version
+    - celery worker --help
+    - celery beat --help
+    - celery multi help
+    - celery shell --help
+    - pytest t/unit      # [py3k]
 
 about:
-    home: http://celeryproject.org
-    license_file: LICENSE
-    license: BSD-3-Clause
-    summary: Distributed Task Queue
-    dev_url: https://github.com/celery/celery
-    doc_url: http://docs.celeryproject.org/en/latest/index.html
+  home: http://celeryproject.org
+  license_file: LICENSE
+  license: BSD-3-Clause
+  summary: Distributed Task Queue
+  dev_url: https://github.com/celery/celery
+  doc_url: http://docs.celeryproject.org/en/latest/index.html
 
 extra:
-    recipe-maintainers:
-        - kwilcox
-        - pmlandwehr
-        - sodre
-        - edetec
```

Comment:
Has @edetec approved being removed as a maintainer?

Reply:
I don't know how active on GitHub is he now. If there's no answer till tomorrow, I'll rever the change.

Reply:
Ooops. Missed this one. We'll fix once we hear from @edetec.

Reply:
I talked to him via IM, but I think we can wait to see if he find some time to confirm here

Reply:
Hi guys, sorry for taking so long to answer.
That is okay for me, remove me as a maintainer.